### PR TITLE
Process output documents schedule

### PR DESCRIPTION
**Pull request by stevenwilkin**
The schedule has to be added manually as it can't be configured via `app.json`

### CODE REVIEW

**Comment by stevenwilkin** [comment]
`Procfile` entry added

**Comment by gjvis** [comment]
:+1:

File: Procfile [hunk 1]
```diff
@@ -1,2 +1,3 @@
 web: bin/puma -C config/puma.rb
 worker: bin/sidekiq -C config/sidekiq.yml
+process_output_documents: rake output_documents:process
```
Comment (stevenwilkin) [inline Procfile]: `rake` or `bin/rake` ?

`bin/puma` and `bin/sidekiq` are Bundler shims whereas `bin/rake` [loads Spring](https://github.com/guidance-guarantee-programme/output/blob/master/bin/rake#L3)

Comment (stevenwilkin) [inline Procfile]: Also, alphabetical ordering @andrewgarner ?

Comment (andrewgarner) [inline Procfile]: Yes please. Likewise for the add ons.

**Pull request merged**